### DISCUSSION TITLE
Trickplay in 0.18.3 beta?

### DISCUSSION THREAD

**Discussion by pavichokche** [jellyfin/jellyfin-androidtv#4232] [General]
I saw the release notes mentioning Trickplay for v18 but I haven't seen anything in the app to enable it, nor have I seen them function. Is there a setting which needs to be enabled somewhere on the server?
My trickplay images are properly scraped (I assume?) and they work in the web browser and Jellyfin Media Player program.

**Comment by nielsvanvelzen** [comment]
You'll need to explicitly enable the feature in the "Developer options" of the app. It's marked experimental and not enabled by default because there are some issues that are currently hard to fix (slow loading/user interface not that great). Once enabled the scrubbing previews (TrickPlay) should just work for all items that have it generated.

**Reply by pavichokche** [reply]
Yeah of course after looking for it twice it's only after I posted this and checking a third time that I found it -_-
THank you so much, it seems to work great for me at least :D
